R/elbo_ss.R

```diff
@@ -34,12 +35,11 @@ get_ER2_ss = function(XtX,Xty,s,var_y,n){
 
 
 #' @title posterior expected loglikelihood for a single effect regression
-#' @param X an n by p matrix of covariates
+#' @param d an p vector of diagonal elements of XtX
```

Comment:
"a" p vector, not "an" p vector.

Reply:
Also in `single_effect_regression_ss` you had it `dXtX`. Do you want to stick to one convention?